Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md

```diff
@@ -1,5 +1,12 @@
 G Suite Auditor is an integration that receives audit logs from G Suite's different applications - admin, drive, calendar, and more.
 
+## Required Permissions:
+In order to use the integration you will need:
+* Admin account.
```

Comment:
Are you sure of this? Normal users cannot interact with the API only admins can?

Reply:
It looks like you need the Admin email address. No password is required